### PR TITLE
Create placeholder HTML widget for missing models

### LINKED ISSUE #1042: Embedder: provide a way to render all known widgets and skip unknown
Otherwise, adding a widget type is a backward incompatible change in `jupyter-js-widgets` or for a widget library.

### PR DESCRIPTION
**Pull request by SylvainCorlay**
Fixes #1042.

## In the notebook:

screen shot 2017-02-06 at 7 20 51 pm

## In a web context:

screen shot 2017-02-06 at 7 23 03 pm

## On Styling:

The table should probably have the new table styling of @ellisonbg but that is out of the scope of this PR IMO.

### CODE REVIEW

File: jupyter-js-widgets/src/manager-base.ts [hunk 1]
```diff
@@ -310,9 +314,22 @@ abstract class ManagerBase<T> {
...
+                model_id: model_id,
+                comm: options.comm,
+            }
+            var widget_model = new HTMLModel({}, modelOptions);
+            widget_model.once('comm:close', function () {
+                delete that._models[model_id];
+            });
+            let placeholder =
+                `<div>
+                  <div>ould not create model:</div>
```
Comment (blink1073) [inline jupyter-js-widgets/src/manager-base.ts]: Could*

Comment (SylvainCorlay) [inline jupyter-js-widgets/src/manager-base.ts]: c'mon  I pushed this 15sec ago 😄

File: jupyter-js-widgets/src/manager-base.ts [hunk 2]
```diff
@@ -310,9 +314,22 @@ abstract class ManagerBase<T> {
...
+                comm: options.comm,
+            }
+            var widget_model = new HTMLModel({}, modelOptions);
+            widget_model.once('comm:close', function () {
+                delete that._models[model_id];
+            });
+            let placeholder =
+                `<div>
+                  <div>Could not create model:</div>
+                  <div> + ${error.message} +  </div>
```
Comment (jasongrout) [inline jupyter-js-widgets/src/manager-base.ts]: We don't need the + signs - can we delete them?

Comment (jasongrout) [inline jupyter-js-widgets/src/manager-base.ts]: Also, can we add model_name, model_module, and model_module_version in the error message to help in debugging?

**Comment by jasongrout** [comment]
I haven't tested this yet, but a read-through of the code looks good to me, modulo some comments.

File: jupyter-js-widgets/src/manager-base.ts (hunk 2 above)
Comment (SylvainCorlay) [inline jupyter-js-widgets/src/manager-base.ts]: OK. pluses are leftovers.

Comment (SylvainCorlay) [inline jupyter-js-widgets/src/manager-base.ts]: Will be using an html table with all the information.

**Comment by SylvainCorlay** [comment]
I replaced the simple div with a more informative table.

**Comment by SylvainCorlay** [comment]
The table should probably have the new table styling of @ellisonbg but that is out of the scope of this PR IMO.

**Pull request merged**